base/workflow/R/runModule.run.write.configs.R

```diff
@@ -13,7 +13,6 @@ runModule.run.write.configs <- function(settings, overwrite = TRUE) {
     }
     return(PEcAn.settings::papply(settings, runModule.run.write.configs, overwrite = FALSE))
   } else if (PEcAn.settings::is.Settings(settings)) {
-    write <- settings$database$bety$write
```

Comment:
Can you explain the intended effect of deleting this line? My first guess is maybe the idea was for `settings$database$bety$write` to no longer get consulted at `run.write.configs` time and instead to have the ensemble and get.parameter.samples steps check it directly? But then I'm not sure why the changes in `run.write.configs` were needed, so I think I'm off-track already.

I ask because this deletion is what's causing the current CI failure -- it leaves `write` undefined in the call to run.write.configs on line 29 below, which then means the references to `write` inside run.write.configs get interpreted as the base function `write()` instead of as a logical value. I can think of several possible fixes ranging from re-adding this line to renaming the argument so it doesn't conflict with a base function name, but without knowing more about the intention I don't know which approach to recommend.

Reply:
Sorry for missing your comment earlier; I was busy with finals at the time. You’re absolutely right—I was trying to skip consulting `settings$database$bety$write`, but that shouldn’t have been done in `run.write.configs`. Your valuable comment has helped me since I was in the middle of troubleshooting the CI failure. Thanks!

Reply:
In that case re-adding this line, together with my suggested change in run.write.configs above that @dlebauer already applied for us, should be what you need to get CI passing 🤞